### DISCUSSION TITLE
Background Image

### DISCUSSION THREAD

**Discussion by iamrodriguez** [PHPOffice/PhpSpreadsheet#2211] [Q&A] [answered]
![bckgExl](https://user-images.githubusercontent.com/87014742/124587786-a7a3c580-de75-11eb-87a2-b5b6d984a4fc.png)

How can we use this Background tool using PhpSpreadsheet?

**Comment by oleibman** [comment, marked as answer]
Support added with PR #3795 in Nov. 2023, which was part of release 2.0.0.
